sources/packages/backend/apps/api/src/services/student-scholastic-standings/_tests_/unit/student-scholastic-standings.service.adjustStudyBreaks.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { StudyBreak } from "@sims/sims-db";
+import {
+  StudentRestrictionService,
+  StudentScholasticStandingsService,
+} from "../../..";
+import {
+  NotificationActionsService,
+  StudentRestrictionSharedService,
+} from "@sims/services";
+import { DataSource } from "typeorm";
+import { createMock } from "@golevelup/ts-jest";
+import { dateDifference } from "@sims/utilities";
+import { OFFERING_STUDY_BREAK_MAX_DAYS } from "../../../../utilities";
+
+class PublicClassToTestAdjustStudyBreak extends StudentScholasticStandingsService {
+  public adjustStudyBreaks(
+    studyBreaks: StudyBreak[],
+    newStudyEndDate: string,
+  ): StudyBreak[] {
+    return super.adjustStudyBreaks(studyBreaks, newStudyEndDate);
+  }
+}
+
+describe("StudentScholasticStandingsService-adjustStudyBreaks", () => {
+  let studyBreaks: StudyBreak[];
+  let publicClassToTestAdjustStudyBreak: PublicClassToTestAdjustStudyBreak;
+
+  beforeAll(() => {
+    publicClassToTestAdjustStudyBreak = new PublicClassToTestAdjustStudyBreak(
+      createMock<DataSource>(),
+      createMock<StudentRestrictionService>(),
+      createMock<NotificationActionsService>(),
+      createMock<StudentRestrictionSharedService>(),
+    );
+    studyBreaks = [
+      {
+        breakDays: 16,
+        breakEndDate: "2023-12-16",
+        breakStartDate: "2023-12-01",
+        eligibleBreakDays: 16,
+        ineligibleBreakDays: 0,
+      },
+    ];
+  });
+
+  it("Should return the same study break when new study end date is greater than the break end date.", () => {
+    // Arrange
+    const newStudyEndDate = "2023-12-30";
+
+    // Act
+    const adjustedStudyBreaks =
+      publicClassToTestAdjustStudyBreak.adjustStudyBreaks(
+        studyBreaks,
+        newStudyEndDate,
+      );
+
+    // Assert
+    expect(adjustedStudyBreaks).toStrictEqual(studyBreaks);
+  });
+
+  it("Should ignore the study break return empty array when new study end date is less than the break start date.", () => {
```

Comment:
Should it have an "and" like: "Should ignore the study break AND return an empty array when the new study end date is less than the break start date."